products/BellHybrid/apps/application-bell-relaxation/model/RelaxationSongsModel.cpp

```diff
@@ -64,15 +53,14 @@ namespace app::relaxation
         if (!sound) {
             return nullptr;
         }
-        auto item =
-            gui::option::RelaxationOption{getTypeFromPath(sound->fileInfo.path, songsRepository->getPathPrefixes()),
-                                          sound->tags.title,
-                                          [=]([[maybe_unused]] gui::Item &item) {
-                                              activateCallback(*sound);
-                                              return true;
-                                          },
-                                          [=]([[maybe_unused]] gui::Item &item) { return true; },
-                                          nullptr};
+        auto item = gui::option::RelaxationOption{getTypeFromPath(sound->fileInfo.path),
+                                                  sound->tags.title,
+                                                  [=]([[maybe_unused]] gui::Item &item) {
+                                                      activateCallback(*sound);
+                                                      return true;
+                                                  },
+                                                  [=]([[maybe_unused]] gui::Item &item) { return true; },
```

Comment:
You don't need to capture anything here.

Reply:
done